src/constants.ts

```diff
@@ -11,15 +11,15 @@ export const copyNumberCat = [
   {value: 0, name: 'NORMAL', color: '#dcdcdc', border: 'transparent'},
   //{value: 1, name: 'Low level amplification', color: '#f4a582'},
   //{value: 2, name: 'High level amplification', color: '#ca0020'},
-  {value: 'null', name: 'Unknown', color: '#FCFCFC', border: '#dcdcdc'}
+  //{value: 'null', name: 'Unknown', color: '#FCFCFC', border: '#dcdcdc'}
 ];
-export const unknownCopyNumberValue: any = copyNumberCat[copyNumberCat.length-1].value;
+export const unknownCopyNumberValue: any = NaN;
```

Comment:
Changed the value of the `unknownCopyNumberValue` from `'null'` to `NaN` to make it compatible with the new tourdino filter. The function `isMissingValue()` from LineUp does not filter out `'null'` values. An alternative option would be to adapt the `isMissingValue()` function.

Reply:
I assume you are referring to PR https://github.com/Caleydo/tourdino/pull/55.

Reply:
Yes